Classes/TBODeveloperOverlayLogger/NSString+RegexMatches.h

```diff
@@ -12,22 +12,7 @@
  */
 @interface NSString (RegexMatches)
 
-/**
- * Returns a substring if the given regular expression matches.
- *
- *  @param regex A regex describing the searched substring.
- *
- *  @return The substring that matches the regex.
- */
-- (NSString *_Nonnull)substringThatMatchesRegex:(NSRegularExpression *_Nonnull)regex;
-
-/**
- * Returns a substring if any of the given regular expressions matches.
- *
- *  @param regexes A NSArray of NSRegularExpressions to search for matching substrings.
- *
- *  @return The substring that matches any of the regexes; If there are more than one match, all matches will be concatenated with a newline.
- */
-- (NSString *_Nonnull)substringThatMatchesAnyRegex:(NSArray <NSRegularExpression *> *_Nonnull)regexes;
+- (BOOL)matchesRegex:(NSRegularExpression *)regex;
```

Comment:
Good Point. I forgot those, because the method is quite self explanatory. But still. We should add some.